tcms/testruns/static/testruns/js/get.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(() => {
+
+    const testRunId = $('#test_run_pk').data('pk')
+
+    $('.set-running-button').on('click', () => {
+        // use the JSON RPC API to set the date to null
+        // TODO: how - null, undefined, '0' not working!
+    })
+    
+    $('.set-finished-button').on('click', () => {
+        const now = new Date()
+
+        // TODO: better way to do this
+        const data = {'stop_date': `${now.getFullYear()}-${now.getMonth()+1}-${now.getDate()}`}
+        jsonRPC('TestRun.update', [testRunId, data], () => location.reload(true))
```

Comment:
This doesn't work for me at all. Clicking the button reloads the page but doesn't change the display. I don't see the finished_at timestamp.

How have you tested it ?

Reply:
the usual way, but at some point i mismatched the jq selectors with the html ids, it should be fine now